lib/matplotlib/artist.py

```diff
@@ -119,6 +143,8 @@ def __init_subclass__(cls):
         cls.set.__qualname__ = f"{cls.__qualname__}.set"
         cls._update_set_signature_and_docstring()
 
+        cls.draw = _prevent_rasterization(cls.draw)
```

Comment:
Put this above L133 to make sure this still gets run even if the conditional wants to short-circuit us.